cl/alerts/models.py

```diff
@@ -106,6 +108,29 @@ def save(self, *args, **kwargs):
             self.secret_key = get_random_string(length=40)
         super().save(*args, **kwargs)
 
+    def alert_type_changed(self) -> None:
```

Comment:
Hey, I was thinking about this new method, and I have a suggestion for the name. How about `validate_alert_type_change` instead of `alert_type_changed`?

I think `validate_alert_type_change` better communicates that the method's purpose is to perform a check and potentially raise an exception if the update isn't valid. It feels more aligned with a validation process.

```suggestion
    def validate_alert_type_change(self) -> None:
```

Reply:
Done!